TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Teleop.java

```diff
@@ -809,7 +866,15 @@ else if( gamepad2_dpad_left_now && !gamepad2_dpad_left_last)
         else if( gamepad2_dpad_down_now && !gamepad2_dpad_down_last)
         {   // Raise lift to LOW junction
             robot.grabberSpinStop();
-            robot.grabberSetTilt( robot.GRABBER_TILT_FRONT_L );
+            if(collectingFromStack)
+            {
+                robot.grabberSetTilt(robot.GRABBER_TILT_GRAB3);
```

Comment:
What eventually sets the grabber tilt to GRABBER_TILT_FRONT_L, the tilt we need to score?